Moggles/ClientApp/src/TogglesList.vue

```diff
@@ -15,7 +15,7 @@
                     :pagination-options="getPaginationOptions"
                     :sort-options="{
                       enabled: true,
-                      initialSortBy: {field: 'toggleName', type: 'asc'}
```

Comment:
are you sure that this change will not break the alphabetical filtering? the permanent FTs need to be at the top and the rest of them ordered alphabetically after the permanent ones.

Reply:
Hello, they will still be sorted alphabetically as well. In the FeatureToggleController the FT's are already being sorted by their name.